src/docfx/build/page/PageBuilder.cs

```diff
@@ -134,6 +134,7 @@ public void Build(ErrorBuilder errors, FilePath file)
 
             var systemMetadataJObject = JsonUtility.ToJObject(systemMetadata);
 
+            outputModel["schema"] = mime.Value;
```

Comment:
Move this line to line 152 in case this key is overridden by other processes.

Reply:
ok